modules/ownKeys.js

```diff
@@ -0,0 +1,16 @@
+import objectKeys from 'object-keys'
+
+const reflectOwnKeysExist =
+  typeof Reflect === 'object' && typeof Reflect.ownKeys === 'function'
+
+const ownKeys = (object) => {
+  if (reflectOwnKeysExist) {
+    return Reflect.ownKeys(object)
+      .filter(key => Object.getOwnPropertyDescriptor(object, key).enumerable)
+  }
```

Comment:
I check the [ES6 compatibility table](https://kangax.github.io/compat-table/es6/) and realize what you said is the reason for the failed test. Thank you again for the useful advice.